brave/brave-core
# Update settings to use Leo controls

Resolves https://github.com/brave/brave-browser/issues/31810

Use Leo controls on the new tab page settings dialog, instead of the old `brave-ui` or custom ones.

Before:
![Background Image](https://github.com/brave/brave-core/assets/7678024/9787f3b9-7a2a-42b6-9ac3-c2e3ae2b0faa)
![Brave Stats](https://github.com/brave/brave-core/assets/7678024/d086912e-1ad3-472d-93e3-764ec6b234d5)
![Top Sites](https://github.com/brave/brave-core/assets/7678024/dc218454-ec06-4be6-81a3-b57f4f6c97f2)
![Clock](https://github.com/brave/brave-core/assets/7678024/95e18c9c-e5c6-4bc5-abf5-5a7c147a9939)
![Cards](https://github.com/brave/brave-core/assets/7678024/a38c8649-d63e-4436-b6b1-85de1c2b87b8)

After:
![Background Image](https://github.com/brave/brave-core/assets/7678024/e839b307-8a6a-41bb-807b-272f02b42d27)
![Brave Stats](https://github.com/brave/brave-core/assets/7678024/d866d798-a00b-49d4-beb3-24a41ff3581a)
![Top Sites](https://github.com/brave/brave-core/assets/7678024/af21cf09-b92e-454f-b422-2bbd32bce6e8)
![Clock](https://github.com/brave/brave-core/assets/7678024/6e30e607-c67e-45e5-a054-141cbc436b88)
![Cards](https://github.com/brave/brave-core/assets/7678024/4f4d461c-44d2-4e70-b510-5098a4febcc1)




## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

All of the following is on the new tab page settings dialog:
1. Check the Show Background images toggle works
2. Check the Show sponsored images toggle works
3. Check the top sites toggle works
4. Check the brave news toggle works
5. Check the clock toggle works
6. Check the clock format dropdown works
7. Check the stats toggle works
8. Check the cards toggle works (need to scroll down)
